server/lib/cleanup/languages/fr.js

```diff
@@ -49,9 +51,134 @@ function clean(sentences) {
       .replace(/\?(?!\s+)/g, '? ')
       .replace(/!(?!\s+)/g, '! ')
 
-      // Final normalization of spaces
-      .replace(/\s+/g, ' ')
+	  //special names and places cleanup  	  
+	  //based on common-voice/CorporaCreator#87
+
+	  //Jean-Paul II
+	  .replace(/Jean-Paul II|Jean Paul II/g, 'Jean-Paul deux')
```

Comment:
It was just the work of Nicolas. ...I think we can remove it. (It was based on the dump file of the CorporaCreator errors, it could explain this 'only 1 entry' list)